### PR TITLE
Accessibility bugfix

### PR DESCRIPTION
**Pull request by singhpritam18**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #Acceesbility Issue Fix

Test URLs:
- Before: https://main--world-bank--aemsites.aem.live/ext/en/home
- After: https://accessibility-bugfix--world-bank--aemsites.aem.page/ext/en/home

### CODE REVIEW

File: blocks/header/header.js [hunk 1]
```diff
@@ -280,6 +300,29 @@ async function changeTrendingData(navSections) {
...
+const setAccessibilityAttrForSearchIcon = (contentWrapper) => {
+  const [iconTag] = [...contentWrapper.children];
+  const iconSpan = iconTag.querySelector('span');
+  iconSpan.setAttribute('role', 'button');
+  iconSpan.setAttribute('aria-label', 'Perform a search query');
+  iconSpan.setAttribute('tabindex', 0);
+};
+
+const closeSearchOnFocusOut = (e) => {
+  const navWrapper = document.querySelector('.nav-wrapper');
```
Comment (mkbansal1) [inline blocks/header/header.js:312]: @singhpritam18 avoid using document.querySelector

File: blocks/header/header.js [hunk 2]
```diff
@@ -207,14 +207,28 @@ function createSearchBox() {
...
     cancelImg.style.cssText = 'display: flex; cursor: pointer;';
     cancelContainer.addEventListener('click', () => {
       closeSearchBox();
     });
+    cancelContainer.addEventListener('keydown', (e) => {
+      if (e.key === 'Enter') {
+        closeSearchBox();
+      } else if (e.key === 'Escape') {
+        closeSearchBox();
+      }
```
Comment (thomsebastin) [inline blocks/header/header.js:230]:
```js
if (e.key === 'Enter' || e.key === 'Escape') {
    closeSearchBox();
}
```

File: blocks/header/header.js (hunk 1 above)
Comment (singhpritam18) [reply blocks/header/header.js]: Done

File: blocks/header/header.js (hunk 2 above)
Comment (singhpritam18) [reply blocks/header/header.js]: Done

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**